src/sentry/static/sentry/app/components/forms/select2Field.jsx

```diff
@@ -48,6 +66,12 @@ class Select2Field extends InputField {
       allowClear: this.props.allowClear,
       allowEmpty: this.props.allowEmpty,
       width: 'element',
+      data: (this.props.choices || []).map(choice => {
+        if (choice.constructor === Array) {
```

Comment:
`Array.isArray`

Reply:
SO suggested this was a faster method, and I was concerned about 'instanceof' for compatibility reasons

Reply:
`Array.isArray` is faster on Chrome 61 at least:
https://jsperf.com/array-isarray-vs-instanceof-vs-constructor-check